File: resources/templates/mocha/testcase.js
	it('should test <% TESTCASE_NAME %> - <% SUMMARY %>' , done => {
		const url = `<% ENDPOINT %>/<% PATH %>`;
		const body = <% BODY %>;

		debug(`<% METHOD %>: ${url}`);
		debug(headers);
		debug(JSON.stringify(body, null, 2));

		superagent.<% METHOD_NAME %>(url)
			.set(headers)
			.send(body)
			.type('<% MEDIATYPE %>')
			.end((err, res) => {
				if (err) {
					error(err);
				}
				<% ASSERTIONS %>
				done();
		});
	});

File: resources/templates/junit/TestCase.java
		@Test
		@Details(description="<% SUMMARY %>")
		public void <% TESTCASE_CLASS_NAME %>() throws Exception {
			String url = String.format("%s/<% PATH %>", <% ENDPOINT %>);
			String body = <% BODY %>;
			Response response = request(new URL(url), "<% METHOD %>", body, headers);
			onError(response);
			<% ASSERTIONS %>
		}